plugin-code/src/main/java/com/trivago/cluecumber/CluecumberReportPlugin.java

```diff
@@ -111,6 +119,11 @@ public void execute() throws CluecumberPluginException {
         logger.info("-----------------------------------------------");
         propertyManager.logProperties();
 
+        if (skip) {
+            getLog().info("skipped");
```

Comment:
This log should be more descriptive. It could otherwise be mistaken for a skip caused by errors.